apps/cli/src/sync/utils.ts

```diff
@@ -24,6 +24,25 @@ export function getDefaultSiteConfig(title?: string): SiteConfig {
   };
 }
 
+export async function getDefaultSiteConfigFromRemote(
+  session: ISession,
+  projectId: string,
+  siteProject: SiteProject,
+): Promise<SiteConfig> {
+  const project = await new Project(session, projectId).get();
+  const remoteSiteConfig = await new RemoteSiteConfig(session, project.id).get();
+  const siteConfig = getDefaultSiteConfig();
+  siteConfig.title = project.data.title;
+  siteConfig.logoText = project.data.title;
+  if (remoteSiteConfig.data.domains) siteConfig.domains = remoteSiteConfig.data.domains;
+  // Add an entry to the nav if it doesn't exist (i.e. empty list is fine)
+  if (!remoteSiteConfig.data.nav) {
```

Comment:
Should this have:
```
else {
  siteConfig.nav = remoteSiteConfig.data.nav
}
```

Reply:
I guess this is the same thing as for `title` etc. We need to do a better job of respecting the remote site config once users have an easy way to update....